### PR TITLE
Added input bar to General Layout 

### PR DESCRIPTION
**Pull request by Parsley-Afroman**
note that there are style toggles that will become functional in the following story

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -95,6 +95,68 @@ nav {
     margin: auto;
 }
 
+
+
+.wordInput {
+    height: 70px;
+    width: 70%;
+    margin: 0 auto;
+    margin-bottom: 25px;
```
Comment (luke-grayland) [inline styles.css:104]: Could this be all in one line (Margin: 0 auto 25px auto)?

File: styles.css [hunk 2]
```diff
@@ -95,6 +95,68 @@ nav {
...
+    font-family: var(--letterFont);
+    width: 30%;
+    border-radius: 0px 5px 5px 0px;
+    border-color: #f4eb47;
+    border-style: solid;
+    display: none;
+}
+/* toggling this class completely removes the section*/
+.outcome {
+    /*display: none;*/
```
Comment (luke-grayland) [inline styles.css:149]: Would it be better to leave this commented in for now and only add (or toggle) the class when it's needed?

**Review by luke-grayland — CHANGES_REQUESTED** [review]

File: styles.css (hunk 1 above)
Comment (Parsley-Afroman) [reply styles.css:104]: Nice one, I updated this

File: styles.css (hunk 2 above)
Comment (Parsley-Afroman) [reply styles.css:149]: If I leave it in then the whole thing will disappear so I left it to aid the functionality

**Review by luke-grayland — APPROVED** [review]

**Pull request merged**